nifty-client/src/main/java/com/facebook/nifty/client/HttpClientConnector.java

```diff
@@ -92,6 +92,9 @@ public ChannelPipeline getPipeline()
                 ChannelPipeline cp = Channels.pipeline();
                 cp.addLast("httpClientCodec", new HttpClientCodec());
                 cp.addLast("chunkAggregator", new HttpChunkAggregator(maxFrameSize));
+                if (clientConfig.sslClientConfiguration() != null) {
```

Comment:
ya same as previous reason,  the timeout thing adds itself to the front of the list as well.